packages/rocketchat-ui-message/message/popup/messagePopup.coffee

```diff
@@ -160,7 +160,7 @@ Template.messagePopup.onCreated ->
 
 	template.records = new ReactiveVar []
 	Tracker.autorun ->
-		if template.data.collection.find?
+		if template.data.collection.findOne?
 			template.data.collection.find().count()
```

Comment:
Ok, so in fact you're only checking if that's an array or a mongo collection? 
It looked strange that you check for `findOne` but then use `find`, that's all. And what does it do `template.data.collection.find().count()`?